tests/settings/test_settings.py

```diff
@@ -18,3 +32,25 @@ def test_invalid_storage() -> None:
         assert False, f"'validate_settings' raised an exception {exc}"
     finally:
         cluster[0]["storage_sets"].remove("non_existing_storage")
+
+
+def test_topics_sync_in_settings_validator() -> None:
+
+    all_settings = build_settings_dict()
+    # Make a copy of the default Kafka topic map from settings
+    default_map = deepcopy(all_settings["KAFKA_TOPIC_MAP"])
+    # Overwrite topic map temporarily to include all defined topic names
+    all_settings["KAFKA_TOPIC_MAP"] = {t.value: {} for t in Topic}
+
+    # Validate settings with the new topic map to check
+    # whether all defined topic names correspond to the
+    # topic names in the settings validator
+    try:
+        validate_settings(all_settings)
+    except Exception:
```

Comment:
we want to catch your new exception type here, right?

Reply:
oops! yes